rosa_architecture/rosa-sts-about-iam-resources.adoc

```diff
@@ -19,6 +22,56 @@ This document provides reference information about the IAM resources that you mu
 
 * For steps to quickly create a ROSA cluster with STS, including the AWS IAM resources, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-a-cluster-quickly[Creating a ROSA cluster with STS quickly].
 * For steps to create a ROSA cluster with STS using customizations, including the AWS IAM resources, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-sts-creating-a-cluster-with-customizations[Creating a ROSA cluster with STS using customizations].
+//
+//
+// Content for OSDOCS-3201 and OSDOCS-3203 starts here.
+//
+// These modules explain concept of the ocm-role IAM role. the process to create that role, and then creating AWS Operator roles and linking those roles to the policies
+//
+//
+
+[id="rosa-sts-ocm-roles-and-permissions_{context}"]
+== {cluster-manager} roles and permissions
+
+// Commenting out until ROSA UI release
+// If you create ROSA clusters by using {cluster-manager-url}, you must have the following AWS IAM roles linked to your AWS account to create and manage the clusters. For more information on linking your IAM roles to your AWS account, see xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-associating-account_rosa-sts-aws-prereqs[Associating your AWS account].
```

Comment:
From ISG: Use "information about" unless you are referring to the location of information such as "the information that is on the server".